adapter/PreciceInterface.c

```diff
@@ -280,14 +279,13 @@ void Precice_WriteCouplingData( SimulationData * sim )
                 getNodeDisplacements( interfaces[i]->nodeIDs, interfaces[i]->numNodes, sim->vold, sim->mt, interfaces[i]->nodeVectorData );
                 precicec_writeBlockVectorData( interfaces[i]->displacementsDataID, interfaces[i]->numNodes, interfaces[i]->preciceNodeIDs, interfaces[i]->nodeVectorData );
                 break;
-	    case DISPLACEMENTDELTAS:
+	    	case DISPLACEMENTDELTAS:
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).